apps/designer/app/designer/shared/fonts-manager/item-menu.tsx

```diff
@@ -70,17 +79,20 @@ const ItemMenu = ({
       </DropdownMenuTrigger>
       <DropdownMenuPortal>
         <DropdownMenuContent align="start">
-          <DropdownMenuItem
-            onClick={(event) => {
-              // Prevent setting the current font to the item.
-              event.stopPropagation();
-            }}
-            onSelect={() => {
-              onDelete();
-            }}
-          >
-            <DeprecatedText2>Delete font</DeprecatedText2>
-          </DropdownMenuItem>
+          <Tooltip side="bottom" content={tooltipContent}>
+            <DropdownMenuItem
+              disabled={deleteDisabled}
+              onClick={(event) => {
+                // Prevent setting the current font to the item.
+                event.stopPropagation();
+              }}
+              onSelect={() => {
+                onDelete();
+              }}
+            >
+              <DeprecatedText2>Delete font</DeprecatedText2>
```

Comment:
if its not too much work, please replace deprecated components with new once as you see them, you tested the UI, you know what's supposed to happen, for you it should be couple of minutes.